src/js/components/List/__tests__/List-test.js

```diff
@@ -1,13 +1,153 @@
 import React from 'react';
 import 'jest-styled-components';
-import { cleanup, render, fireEvent } from '@testing-library/react';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
 
+import { cleanup, render, fireEvent } from '@testing-library/react';
+import { axe } from 'jest-axe';
 import { Grommet } from '../../Grommet';
 import { List } from '..';
 
 describe('List', () => {
   afterEach(cleanup);
 
+  test('list should have no violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <List />
+      </Grommet>,
+    );
+
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  test('mouse events', () => {
+    const onClickItem = jest.fn();
+    const { container, getByText } = render(
+      <Grommet>
+        <List
+          data={[{ a: 'alpha' }, { a: 'beta' }]}
+          onClickItem={onClickItem}
+        />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.mouseOver(getByText('beta'));
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.mouseOut(getByText('beta'));
+    expect(container.firstChild).toMatchSnapshot();
+    expect(onClickItem).toBeCalledTimes(0);
+  });
+
+  test('focus and blur', () => {
+    const onClickItem = jest.fn();
+    const { container, getByText } = render(
+      <Grommet>
+        <List
+          data={[{ a: 'alpha' }, { a: 'beta' }]}
+          onClickItem={onClickItem}
+        />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.focus(getByText('beta'));
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.blur(getByText('beta'));
+    expect(container.firstChild).toMatchSnapshot();
+    expect(onClickItem).toBeCalledTimes(0);
+  });
+
+  test('ArrowDown key', () => {
+    const onClickItem = jest.fn();
+    const { container, getByText } = render(
+      <Grommet>
+        <List
+          data={[{ a: 'alpha' }, { a: 'beta' }]}
+          onClickItem={onClickItem}
+        />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('alpha'));
+    fireEvent.mouseOver(getByText('alpha'));
+    fireEvent.keyDown(getByText('alpha'), {
+      key: 'ArrowDown',
+      keyCode: 40,
+      which: 40,
+    });
```

Comment:
```suggestion
    });
    expect(OnClickItem).toHaveBeenCalled();
```
I think that tis would be a useful thing to add here. Look at this throughout the file too

Reply:
Went back and added `expect(OnClickItem).toHaveBeenCalled()` to relevant tests